docs/cyber.md

```diff
@@ -0,0 +1,80 @@
+# access anomalies: [complement_access.py](../src/main/python/mmlspark/cyber/ml/access_anomalies/complement_access.py)
+- [Talk at European Spark Conference 2019](https://databricks.com/session_eu19/cybermltoolkit-anomaly-detection-as-a-scalable-generic-service-over-apache-spark)
+- [(Internal Microsoft) Talk at MLADS November 2018](https://resnet.microsoft.com/video/42395)
+- [(Internal Microsoft) Talk at MLADS June 2019](https://resnet.microsoft.com/video/43618)
+
+1. [ComplementAccessTransformer](../src/main/python/mmlspark/cyber/ml/access_anomalies/complement_access.py)
+   is a SparkML [Transformer](https://spark.apache.org/docs/2.2.0/api/java/index.html?org/apache/spark/ml/Transformer.html).
+   Given a dataframe it returns a new dataframe containing new access patterns sampled from
+   the set of possible access patterns which did not occur in the given dataframe
+   (i.e., it returns a sample from the complement set).
+
+# feature engineering: [indexers.py](../src/main/python/mmlspark/cyber/ml/feature_engineering/indexers.py)
```

Comment:
these seem really useful! Is there a reason to put these (feature engineering PipelineStages) under the cyber ml namespace specifically? The descriptions suggest they are more generally useful

Reply:
I think we should consider putting these utility type transformers in the pipeline stages section of the main scala repo. WIll be easier to maintain them there and will get some test coverage for free. What are your thoughts @rolevin?

Reply:
I guess they are more generic ... Note however that the indexers and scalars are useful for handling a situation in which the data belongs to multiple tenants, each having his own unique id. Where exactly would you suggest I move this package?